scripts/backend_tests.py

```diff
@@ -239,6 +242,15 @@ def _convert_to_test_target(path):
 
 def main():
     """Run the tests."""
+
+    # Run the tests for explicit_kwargs_checker.
```

Comment:
So this is actually what I _didn't_ want to do. Ideally there shouldn't be special-case logic for the checker test.

Could you structure things in such a way such that `bash scripts/run_backend_tests.sh --test_target=scripts.explicit_kwargs_checker_test` or `bash scripts/run_backend_tests.sh --test_path=scripts`  works, just like it would for any other test file?

Reply:
Got it. I'll make the necessary changes. Thanks!

Reply:
@seanlip: I'm facing a problem here. For our tests, we use the class `TestBase` which inherits the `unittest.TestCase` class. For the `ExplicitKwargsCheckerTest` class, I am inheriting the `pylint.testutils.CheckerTestCase` class. Therefore I could not find a possible way to integrate it with the unittests.  When I try to link it, it says,"'No tests found". I looked into the `CheckerTestCase` class as well.  
I suggest we write a separate bash script, `run_custom_lint_tests.sh` and call the `explicit_kwargs_checker_test.py` from that scripts. We could then add the bash script to `run_tests.sh`. Please suggest!  
Thanks!

Reply:
Welcome back!

There's only one criterion: adding a new lint test should not require any additional script-level configuration in the future (i.e. autodiscovery should be supported). Any solution that satisfies that is fine. I don't think the solution you suggest satisfies that though.

There are two approaches I can think of. One is multiple inheritance. The other is something similar to the code in core/domain/interaction_registry.py that automatically scans a subdirectory in the codebase and extracts all the classes with a particular property.

I lean towards implementing the second one in scripts/backend_tests.py.